makefile

```diff
@@ -86,6 +86,7 @@ gen-jaxrs: api.raml merge-raml
 	@$(BIN_DIR)/generate-jaxrs.sh $(GEN_PACKAGE)
 	@$(BIN_DIR)/generate-jaxrs-streams.sh $(GEN_PACKAGE)
 	@$(BIN_DIR)/generate-jaxrs-postgen-mods.sh $(GEN_PACKAGE)
+	@grep -Rl javax src | xargs -I{} sed -i 's/javax.ws/jakarta.ws/g' {}
```

Comment:
Can this line live in generate-jaxrs-postgen-mods.sh in the jersey-3 branch?  Seems low level to have to put in all the makefiles.

Reply:
I like this idea, I decided to go with your other suggestion to keep build-utils on a single branch. It'd be nice if the code generation could do this.